### DISCUSSION TITLE
JSBSim in QT

### DISCUSSION THREAD

**Discussion by plmqwerty** [JSBSim-Team/jsbsim#1201] [General]
Can I use JSBSim.dll or JSBSim.lib in qt creator??
I created these lib files and add in my project (qt5.15) but I encountered many errors 
`qt unresolved external symbol __declspec(dllimport) lib file`
Anyone can guide me!!

**Comment by bcoconni** [comment]
If you're compiling your code with `JSBSim.lib` i.e. static linking, you should define the flag `JSBSIM_STATIC_LINK` for your compiler.

Regarding the error message that you're mentioning, it would be helpful if you could provide the symbol (i.e. class, method or function name) that is triggering this error message: not all JSBSim classes are exported to the DLL so if you try to access one of them, the compiler will indeed complain.
